packages/fetchai/skills/erc1155_client/tests/test_behaviours.py

```diff
@@ -52,7 +52,8 @@ def test_setup(self):
     def test_act_is_searching(self):
         """Test the act method of the search behaviour where is_searching is True."""
         # setup
-        self.strategy._is_searching = True
+        self.strategy.is_searching = True
+        assert self.strategy.is_searching
```

Comment:
Looks unrelated, why the change?

Reply:
see comment above: https://github.com/valory-xyz/open-aea/pull/308#discussion_r968856718